examples/platform/k32w/app/support/Memconfig.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#ifndef SRC_PLATFORM_K32W_MEMCONFIG_H_
+#define SRC_PLATFORM_K32W_MEMCONFIG_H_
+
+namespace chip {
```

Comment:
I don't think namespaces are needed if you "extern C"

Reply:
Same for above Memconfig.cpp - namespaces should not be needed.

Reply:
Thank you for your observations, @andy31415. I included them in my last commit.